sdks/swagger/sdk.swagger.json

```diff
@@ -443,20 +420,33 @@
         }
       },
       "title": "Key, Value entry"
-    }
-  },
-  "x-stream-definitions": {
-    "sdkGameServer": {
+    },
+    "googlerpcStatus": {
       "type": "object",
       "properties": {
-        "result": {
-          "$ref": "#/definitions/sdkGameServer"
+        "code": {
+          "type": "integer",
+          "format": "int32"
+        },
+        "message": {
+          "type": "string"
         },
-        "error": {
-          "$ref": "#/definitions/runtimeStreamError"
+        "details": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/protobufAny"
+          }
+        }
+      }
+    },
+    "protobufAny": {
+      "type": "object",
+      "properties": {
+        "@type": {
+          "type": "string"
         }
       },
-      "title": "Stream result of sdkGameServer"
+      "additionalProperties": {}
```

Comment:
This is workaround. Add jq logic to sdks/swagger/sdk.swagger.json because protoc-gen-openapiv2 doesn't work well in Stream and doesn't generate nessesary definitions